package.json

```diff
@@ -9,5 +9,8 @@
     "bugs": {
         "url": "https://github.com/Doist/prettier-config/issues"
     },
-    "homepage": "https://github.com/Doist/prettier-config#readme"
+    "homepage": "https://github.com/Doist/prettier-config#readme",
+    "peerDependencies": {
+        "prettier": "^1.0.0"
```

Comment:
I'd argue we want to have `^2.0.0` aka latest

Reply:
Of course, it's a mistake.